jobserv/api/worker.py

```diff
@@ -29,6 +31,12 @@ def _is_worker_authenticated(worker: Worker):
         parts = key.split(" ")
         if len(parts) == 2 and parts[0] == "Token":
             return worker.validate_api_key(parts[1])
+        if len(parts) == 2 and parts[0] == "Bearer":
+            try:
+                w = worker_from_jwt(parts[1])
+                return w.name == worker.name
```

Comment:
So, each worker requires a dedicated JWT token?

Reply:
yes. and scaling logic (or perhaps another company that wanted to privately host their own ci workers) - can create the tokens as needed: https://github.com/foundriesio/rftm/blob/main/rfds/0006/launch.py

Reply:
How to guarantee a worker name uniqueness?

Reply:
the DB will reject it and an error will get thrown.